hudi-common/src/main/java/org/apache/hudi/common/table/HoodieTableMetaClient.java

```diff
@@ -821,6 +820,8 @@ public static class PropertyBuilder {
     private String metadataPartitions;
     private String inflightMetadataPartitions;
     private String secondaryIndexesMetadata;
+    private Boolean multipleBaseFileFormatsEnabled;
+    private String baseFileFormats;
```

Comment:
Can you explain in high level why we need this config `baseFileFormats` ? And why it must a table config here, actually we can merge these two variables into one, for exmaple, an empty string of base formats represent `disabled`.

Reply:
Actually we may not need both the configs. Let me think over.

Reply:
removed.. only keeping the boolean config.